src/Pattern/ObjectCache.php

```diff
@@ -200,10 +212,13 @@ public function __call($method, array $args)
      * is enabled and the property doesn't exist in real. If so it calls __set
      * and removes cached data of previous __get and __isset calls.
      *
+     * @see    http://php.net/manual/language.oop5.overloading.php#language.oop5.overloading.members
+     *
+     * @phpcs:disable Squiz.Commenting.FunctionComment.InvalidReturnVoid
```

Comment:
Had to add this as some upstream project might added fluent interface stuff to the `call` method.

I'll remove this in a dedicated PR for 3.0.